src/ewmh_m2m/__main__.py

```diff
@@ -56,7 +56,7 @@ def main():
         action="store",
         type=Ordinal.get,
         choices=list(Ordinal),
-        default=Ordinal.EAST.name.capitalize(),
+        default=Ordinal.NEXT.name.capitalize(),
```

Comment:
I don’t feel comfortable to change such default value. I would require anybody using the script without argument to update their configuration (in case the new value does not exhibit the same behavior).

Reply:
You're right, I was thinking about this change, my main argument on this change is that after downloading your script, it's required to read help and decide which option should be used. But for current users, this change can be annoying, right.